src/langs/en.json

```
@@ -106,5 +106,10 @@
   "MEMBER_STORAGE_PARENT_FOLDER": "Parent folder",
   "NO_PARENT": "-",
   "STORAGE_FILES_ERROR": "Oops, an error occurred while fetching storage files. Please try again later.",
-  "STORAGE_FILES_EMPTY": "You didn't upload any file."
+  "STORAGE_FILES_EMPTY": "You didn't upload any file.",
+  "EXPORT_INFORMATIONS_TITLE": "Export data",
+  "EXPORT_INFORMATIONS_DESCRIPTION": "You can download a copy of your data",
+  "EXPORT_INFORMATIONS_BUTTON_TEXT": "Start exporting",
```

I'm wondering if `Start exporting` is the best choice here. I'd write something like the following. I don't know what do you think and what the other thinks ?

```suggestion
  "EXPORT_INFORMATIONS_BUTTON_TEXT": "Request your data",
```

---

```suggestion
  "EXPORT_INFORMATIONS_BUTTON_TEXT": "Download your data",
```